.circleci/config.yml

```diff
@@ -151,5 +255,26 @@ workflows:
             parameters:
               py_version: ["3.6", "3.7", "3.8"]
 
+
+  plugin_tests:
+    when: << pipeline.parameters.plugin_test >>
+    jobs:
+      - test_plugin_linux:
+          matrix:
+            parameters:
+              py_version: ["3.6", "3.7", "3.8"]
+              test_plugin: [<< pipeline.parameters.plugin >>]
+      - test_plugin_macos:
+          matrix:
+            parameters:
+              py_version: ["3.6", "3.7", "3.8"]
+              test_plugin: [<< pipeline.parameters.plugin >>]
+      - test_plugin_win:
+          matrix:
+            parameters:
+              py_version: ["3.6", "3.7", "3.8"]
+              test_plugin: [<< pipeline.parameters.plugin >>]
```

Comment:
should this be test_plugins?

Reply:
do you mean re-naming this param?

Reply:
yes. this is not a plugin right? it's a list of plugins.

Reply:
good point. fixed.